forwards_paper.tex

```diff
@@ -307,15 +312,15 @@ \subsection*{Working with a tree sequence through tables}
 of a single branch that occurs in all trees covering this interval.
 The spatial extent of the edges defining the topology of Figure~\ref{fig:example_tree_sequence}
 are shown in the bottom left panel.
-For example, the branch joining nodes 1 to 3 appears both trees,
+For example, the branch joining nodes 1 to 3 appears in both trees,
 and so is recorded as a single edge extending over the whole chromosome.
 It is this method of capturing the shared structure between adjacent trees that makes the
 tree sequence encoding very compact and algorithmically efficient.
 
 Recovering the sequence of trees from this information is straightforward:
 each point along the genome that the tree topology changes
-is accompanied by the end of some \emph{edges} and the beginning of others.
-Since each \emph{edge} records the genomic interval
+is accompanied by the end of some {edges} and the beginning of others.
+Since each {edge} records the genomic interval
```

Comment:
By this point I don't think we need to be saying `edge` every time we talk about an edge.

Reply:
Agreed. I think we're overdoing the \emph a little bit at the moment.